es-spatial-plugin/test/cmr/elasticsearch/plugins/spatial/plugin_test.clj

```diff
@@ -0,0 +1,5 @@
+(ns cmr.elasticsearch.plugins.spatial.plugin-test
+  (:require
+   [clojure.test :refer :all]))
+
+(deftest ^:kaocha/pending spatial-plugin-test)
```

Comment:
Maybe add a comment about why this file is here. It's hard to infer based on the contents of the file